README.rst

```diff
@@ -122,6 +122,7 @@ Optionally, you can restore a sample dataset. For that you need to download the
 
 Second way. Create a database from the sample dump using ``pg_restore``
 ```````````````````````````````````````````````````````````````````````
+QWAT can be tested by downloading the QGS project repository `here <https://github.com/qwat/QWAT>`_ and restoring the latest `data_and_structure_sample.backup <https://github.com/qwat/qwat-data-model/releases/latest>`_ in PostgreSQL (+ extension PostGIS and hstore).
```

Comment:
I don't find this very clear. Does that mean the full QWAT Git repo should be cloned? Or does it mean that only the QGIS project (the `.qgs` file) should be downloaded?

Also, I'd remove `(+ extension PostGIS and hstore)`, which is a bit confusing in my opinion.

Reply:
For now, you have to download the repo (git or zip) to open the QGS project.
Maybe we should create a dedicated repository for that?